docs/getting-started.md

```diff
@@ -106,7 +108,9 @@ module.exports = [{
         },
         { loader: 'extract-loader' },
         { loader: 'css-loader' },
+        { loader: 'postcss-loader' },
```

Comment:
If you attempt to run `npm start` with this configured as-is at this step, you get an error because PostCSS can't find a config. Perhaps we should leave the initial webpack setup as-is and simply add `postcss-loader` when we need it below.